src/libraries/Microsoft.PowerFx.Core/Parser/TexlParser.cs

```diff
@@ -1766,11 +1766,12 @@ internal static string GetTokString(TokKind kind)
             }
         }
 
-        public static string Format(string text)
+        public static string Format(string text, Flags? flags = null)

```

Comment:
This should probably have a similar signature to ParseSecipt above in this file, which uses `Flags flags = Flags.None` above.  We don't generally use nullable types.